boards.txt

```diff
@@ -24623,6 +24623,230 @@ nebulas3.menu.EraseFlash.all=Enabled
+lionbits3.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+lionbits3.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+lionbits3.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+lionbits3.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+lionbits3.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+lionbits3.menu.PartitionScheme.fatflash.build.partitions=ffat
+lionbits3.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+lionbits3.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
+lionbits3.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+lionbits3.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+lionbits3.menu.PartitionScheme.rainmaker=RainMaker
+lionbits3.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
+lionbits3.menu.PartitionScheme.rainmaker.upload.maximum_size=3145728
+
+lionbits3.menu.CPUFreq.240=240MHz (WiFi)
+lionbits3.menu.CPUFreq.240.build.f_cpu=240000000L
+lionbits3.menu.CPUFreq.160=160MHz (WiFi)
+lionbits3.menu.CPUFreq.160.build.f_cpu=160000000L
+lionbits3.menu.CPUFreq.80=80MHz (WiFi)
+lionbits3.menu.CPUFreq.80.build.f_cpu=80000000L
+lionbits3.menu.CPUFreq.40=40MHz
+lionbits3.menu.CPUFreq.40.build.f_cpu=40000000L
+lionbits3.menu.CPUFreq.20=20MHz
+lionbits3.menu.CPUFreq.20.build.f_cpu=20000000L
+lionbits3.menu.CPUFreq.10=10MHz
+lionbits3.menu.CPUFreq.10.build.f_cpu=10000000L
+
+lionbits3.menu.UploadSpeed.921600=921600
+lionbits3.menu.UploadSpeed.921600.upload.speed=921600
+lionbits3.menu.UploadSpeed.115200=115200
+lionbits3.menu.UploadSpeed.115200.upload.speed=115200
+lionbits3.menu.UploadSpeed.256000.windows=256000
+lionbits3.menu.UploadSpeed.256000.upload.speed=256000
+lionbits3.menu.UploadSpeed.230400.windows.upload.speed=256000
+lionbits3.menu.UploadSpeed.230400=230400
+lionbits3.menu.UploadSpeed.230400.upload.speed=230400
+lionbits3.menu.UploadSpeed.460800.linux=460800
+lionbits3.menu.UploadSpeed.460800.macosx=460800
+lionbits3.menu.UploadSpeed.460800.upload.speed=460800
+lionbits3.menu.UploadSpeed.512000.windows=512000
+lionbits3.menu.UploadSpeed.512000.upload.speed=512000
+
+lionbits3.menu.DebugLevel.none=None
+lionbits3.menu.DebugLevel.none.build.code_debug=0
+lionbits3.menu.DebugLevel.error=Error
+lionbits3.menu.DebugLevel.error.build.code_debug=1
+lionbits3.menu.DebugLevel.warn=Warn
+lionbits3.menu.DebugLevel.warn.build.code_debug=2
+lionbits3.menu.DebugLevel.info=Info
+lionbits3.menu.DebugLevel.info.build.code_debug=3
+lionbits3.menu.DebugLevel.debug=Debug
+lionbits3.menu.DebugLevel.debug.build.code_debug=4
+lionbits3.menu.DebugLevel.verbose=Verbose
+lionbits3.menu.DebugLevel.verbose.build.code_debug=5
+
+lionbits3.menu.EraseFlash.none=Disabled
+lionbits3.menu.EraseFlash.none.upload.erase_cmd=
+lionbits3.menu.EraseFlash.all=Enabled
+lionbits3.menu.EraseFlash.all.upload.erase_cmd=-e
+
+##############################################################
+=======
```

Comment:
Remove this. It's some leftover from the conflicts.
```suggestion
```

Reply:
resolved